src/frontend/js/index.js

```diff
@@ -334,7 +334,7 @@ const mapboxInit = ({lng, lat, zoom}) => {
             },
             paint: {
                 'line-opacity': .8,
-                'line-color': 'red',
+                'line-color': 'hsl(283, 100%, 32%)',
```

Comment:
Since we use this twice, we should probably store it as a constant so it only needs to be changed in one place to change everything that is the same colour.

Reply:
Makes sense, but since the hue and saturation values are variable I've turned it into a function instead.